src/test/java/org/isf/generaldata/GeneralDataTest.java

```diff
@@ -0,0 +1,22 @@
+package org.isf.generaldata;
+
+import static org.junit.Assert.*;
+
+import org.junit.Ignore;
+import org.junit.Test;
+
+public class GeneralDataTest {
+
+	@Ignore
+	@Test
+	public void testGetGeneralData() {
+		GeneralData generalData =  GeneralData.getGeneralData();
+		
+		assertNotNull(generalData);
+		
+		assertEquals("es",GeneralData.LANGUAGE);
```

Comment:
and this becomes `assertThat(GeneralData.LANGUAGE).isEqualTo("en")`

Reply:
Fixed